frontend/views/containers/payments/PaymentNextDistributionPill.vue

```diff
@@ -18,8 +18,13 @@ export default ({
     ]),
     nextDistributionDateShort () {
       const nextDisDate = this.dueDateForPeriod(this.currentPaymentPeriod)
-
-      return humanDate(nextDisDate, { month: 'short', day: 'numeric' })
+      const localeTimeString = humanDate(nextDisDate, {
+        hour: '2-digit',
+        minute: '2-digit',
+        hour12: false
+      }, false)
+      const localeDateString = humanDate(nextDisDate, { month: 'short', day: 'numeric' })
+      return `${localeTimeString} - ${localeDateString}`
```

Comment:
It is much simpler to just pass these options without modifying how `humanDate` is implemented:

```js
return humanDate(nextDisDate, { month: 'short', day: 'numeric', hour: 'numeric', minute: '2-digit' })
```

You also do not need to use `hour12`, because that should be handled correctly by `locale` value inside of `humanDate`.

For example:

```
> new Date().toLocaleDateString('en-US', { month: 'short', day: 'numeric', hour: 'numeric', minute: '2-digit' })
'Apr 12, 4:50 PM'
> new Date().toLocaleDateString('ja', { month: 'short', day: 'numeric', hour: 'numeric', minute: '2-digit' })
'4月12日 16:50'
```

Reply:
Yeah. I want to do this way. But the problem is that I don't think we can make the same format using `toLocaleDateString`.
That's why I updated the `humanDate` function and make it possibly return only time.
![image](https://user-images.githubusercontent.com/57976479/231612849-6027d62f-ff35-4e84-a30e-b1e6756dcbe2.png)

Reply:
Either way, in that case you still do not need to modify `humanDate`, you can just call it twice with different `options`. But in that case, still `hour12` should not be passed, it should always be decided by the locale.

However, I think the way it formats `{ month: 'short', day: 'numeric', hour: 'numeric', minute: '2-digit' }` right now is perfectly acceptable, and by calling it once with those options, we are ensured that it will always do it in a locale-appropriate way, whereas creating our own string use ` - ` might not be appropriate for some locales

Reply:
Make sense. I should make that dateTime string. And split and merge again to get the right format.
Not sure why I didn't think this way. Hahaha